src/components/LoadingSpinner.tsx

```
@@ -0,0 +1,15 @@
+export default function LoadingSpinner({ message }: { message?: string }) {
+  return (
+    <div className="flex-col-center h-full gap-6">
+      <div className="relative">
+        <div className="h-12 w-12 animate-spin rounded-full border-4 border-purple-100"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)